docs/src/devguide/getting_started.rst

```diff
@@ -6,13 +6,19 @@ Getting started working on Cython
 Git
 ---
 
-Cython is current hosted in `on GitHub <https://github.com/cython/cython/>`_ and uses Git to track changes to
+Cython is currently hosted `on GitHub <https://github.com/cython/cython/>`_ and uses Git to track changes to
 the source code.  The majority of development happens on the "master" branch, but we typically maintain one or
 two stable release branches which are updated with bugfixes.  Changes that go into the main Cython repository should
 never make the branch unreleasable (even if minor breakages can happen from time to time).
 
-To get started working with Cython you'd usually clone the main Github repo (so that it appears on
-``github.com/<your_user_name>/cython``) and then clone that onto your local PC using:
+To get started working with Cython's code, you will require:
+
+* a `GitHub account <https://github.com/signup>`_
+* `git <https://git-scm.com/downloads>`_
+
+Then you will `fork the code <https://github.com/cython/cython/fork>`_, creating a repository in your own account named ``github.com/<your_user_name>/cython``.
```

Comment:
People who want to work on Cython's code base do not necessarily need a Github account and I can understand that some people don't want to sign up with them. I don't feel like we should promote it. I would guess that anyone who feels a need to work on Cython's code base would probably also have heard of Github and won't need an introduction specifically from our side.